docs/spl-docs/sort-command.md

```diff
@@ -0,0 +1,94 @@
+
+## Syntax
+The required syntax is in bold
+
+**sort**
+[\<count\>]
+**\<sort-by-clause\>...**
+[desc]
+
+### Required Arguments
+
+#### \<sort-by-clause\>
+**Syntax:** `[ - | + ] <sort-field>, ( - | + ) <sort-field> ...`\
+**Description:** Specifies the fields to sort by, including the order of sorting. A minus sign (-) indicates descending order, while a plus sign (+) denotes ascending order. For multiple fields, separate each with a comma.
+
+### Optional Arguments
+
+#### \<count\>
+**Syntax:** `<int> | limit=<int>`\
+**Description:** Determines the number of results to display after sorting. Absent a specified count, a default threshold of 10000 results is applied. Specifying 0 returns all possible results. The count can be directly indicated by an integer or introduced with the prefix 'limit=', e.g., limit=10.
+
+**Default:** 10000
+
+#### desc
+**Syntax:** `d | desc`\
+**Description:** Inverts the sequence of the results. When multiple fields are designated, it inversely arranges the values within those fields according to their specified sequence. For instance, with three fields indicated, the `desc` parameter inversely organizes the values in the first field. For any group of identical values in the first field, it similarly inverts the order of related values in the second field, and for each group of matching values in the second field, it reverses the order of associated values in the third field.
+
+### Sort Field Options
+
+#### \<sort-field\>
+**Syntax:** `<field> | auto(<field>) | str(<field>) | ip(<field>) | num(<field>)`\
+**Description:** Defines the options available for `<sort-field>`.
+
+#### \<field\>
+**Syntax:** `<string>`\
+**Description:** Identifies the field to be sorted.
+
+#### auto
+**Syntax:** `auto(<field>)`\
+**Description:** Automatically determines the best method to sort the field values.
+
+#### ip
+**Syntax:** `ip(<field>)`\
+**Description:** Treats the field values as IP addresses for sorting.
+
+#### num
+**Syntax:** `num(<field>)`\
+**Description:** Considers the field values as numbers for sorting purposes.
+
+#### str
+**Syntax:** `str(<field>)`\
+**Description:** Sorts the field values alphabetically, treating them as strings.
+
+## Examples
+
+1. **Use the sort field options to specify field types**
+Sort the results by the `http_status` field in ascending order, and then sort by `latency` in descending order to see the slowest responses at the top for each status code.
+
+    ```spl
+    ... | sort num(http_status), -num(latency)
+    ```
+
+2. **Specifying the number of results to sort**
+Sort first 100 results in descending order of the `latency` field and then by the `city` value in ascending order. This example specifies the type of data in each of the fields. The `latency` field contains numbers and the `city` field contains strings.
```

Comment:
I think you meant for the description of the example to render on the line after the title of the example. If so, you need to add another empty line (e.g., after line 64)
<img width="1010" alt="image" src="https://github.com/user-attachments/assets/44b3b9d3-a4ae-49bb-ba04-164becf2cdbb">